test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +55,12 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+    
+    private void assertAnyNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
+    }
+    
+    private void assertNotNull(Object... objects) {
+        assertFalse(Utils.isAnyNull(objects));
+    }
```

Comment:
Do note that it is not necessary to abstract the `assertTrue` and `assertFalse` as separate methods (it was done previously to avoid having to type out `Arrays.asList` each time). However, it is also considered good practice to follow existing coding styles.